internal/graph/graph.go

```diff
@@ -52,6 +55,8 @@ type ResolutionMetadata struct {
 	DatastoreQueryCount uint32
 
 	// The number of Check subproblems that had to be dispatched to resolve the parent subproblem this response is associated with.
+	// Thinking of a Check as a tree of evaluations:
```

Comment:
```suggestion
	// Thinking of a Check as a tree of evaluations, DispatchCount will be the sum of all of the subproblems the parent subproblem has to evaluate.
```

Let's keep using already coined terms. We define subproblems above. Node isn't too clear in this context.

Reply:
I feel like tree & node go well together, since they belong to graph theory

Reply:
IMO both terms have their place in the docs. I do think "node" specifically fits when talking about graphs in the abstract. Also might be good to connect nodes and subproblems when defining subproblems.